docs/foundations/color/usage.md

```diff
@@ -12,228 +12,179 @@ subnav:
 </script>
 
 <link rel="stylesheet"
       href="/assets/packages/@rhds/elements/elements/rh-table/rh-table-lightdom.css"
       data-helmet>
 
+<style>
+  td ul {
+    margin-block: 0;
+  }
+  figcaption { 
+    margin-block-start: var(--rh-space-lg);
+    color: var(--rh-color-text-secondary-on-light);
+    font-size: var(--rh-font-size-body-text-sm);
+  }
+  figure uxdot-example {
+    margin-block-start: var(--rh-space-2xl);
+  }
+</style>
+
 ## Our color palette
 
 Our color palette was built to be extensive and flexible. Applying color 
 thoughtfully and consistently helps us deliver accessible, cohesive, and 
 recognizable experiences for our users.
 
-[Contact us][contactus] if you have design system questions or [visit the brand 
-standards page][brandstandards] if you have brand questions.
+[Contact us][contactus] if you have design system questions or visit the [Brand 
+standards website][brandstandards] if you have brand questions.
 
-### Brand red
+## Brand red
 
-Red is our primary brand color. Red is also a strong color, so **use it as an 
-accent**, not to fill large areas.
+Red is our primary brand color. Red is also a strong color, so **use it as an accent**, not to fill large areas.
 
 <uxdot-example width-adjustment="948px">
-  <img src="/assets/foundations/foundations-color-usage-brand-red.svg" 
-      alt="spectrum of brand red shades with examples of brand red being used"
-      width="948px"
-      height="300px">
+  <img alt="Examples of brand red being used in logos, illustrations, and text"
+       src="/assets/color/color-usage-brand-red.svg">
 </uxdot-example>
 
 ### Red orange
 
 For danger or error states, use `red orange`.
 
+<uxdot-example width-adjustment="652px">
+  <img alt="Examples of red orange used in alert, buttons, tag, and site status"
+       src="/assets/color/color-usage-red-orange.svg">
+</uxdot-example>
+
 ## Backgrounds
 
 ### Canvas
 
-Canvas is the bottom-most infinite container that holds all containers, 
-elements, etc. The default canvas colors for light and dark themes are `white` 
-and `black` respectively, but some gray colors can also be used.
+Canvas is the bottom-most infinite container that holds all containers, elements, etc. The default canvas colors for light and dark themes are white and black respectively, but some gray colors can also be used.
 
-For more bespoke websites or interfaces, other colors may be used as long as 
-color contrast and other accessibility requirements are maintained.
+For more bespoke websites or interfaces, other colors may be used as long as color contrast and other accessibility requirements are maintained.
 
-<rh-alert state="note">
+<rh-alert state="info">
   <h4 slot="header">Helpful tip</h4>
-  <p>Elements like Card and other containers include more colors and can be 
-     placed on top of a canvas.</p>
+  <p>Components like Card and other containers include more colors and can be placed on top of a canvas.</p>
```

Comment:
use "Elements" instead of "Components". We spent an entire quarter on this, so let's not regress

Reply:
Thanks for catching that, @bennypowers! I've replaced any uses of "component(s)."